client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/style.scss

```diff
@@ -1,3 +1,4 @@
+@import "@automattic/typography/styles/variables";
```

Comment:
I added this since a variable (`$sans` in line 11) became undefined in the Storybook build.
(This does not affect Calypso because `@import`ed styles should not be built in duplicate, I believe.)